contrib/endpoints/src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -699,12 +699,16 @@ grpc_jwt_verifier_status JwtValidatorImpl::FillUserInfoAndSetExp(
 
   // Optional field.
   const grpc_json *grpc_json = grpc_jwt_claims_json(claims_);
+
+  char *json_str =
```

Comment:
Thank you. Missed this while addressing review comments. I was freeing this in the first commit :-)